source/renderer/app/components/voting/voting-info/RegisterToVote.messages.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
```

Comment:
@lucas-barros please remove this blank line 🙏

Reply:
Updated, thanks!